openmls/src/group/core_group/mod.rs

```diff
@@ -117,11 +123,13 @@ impl CoreGroupBuilder {
         self
     }
     /// Set the [`Vec<PreSharedKeyId>`] of the [`CoreGroup`].
+    #[cfg(any(feature = "test-utils", test))]
     pub fn with_psk(mut self, psk_ids: Vec<PreSharedKeyId>) -> Self {
         self.psk_ids = psk_ids;
         self
     }
     /// Set the [`ProtocolVersion`] of the [`CoreGroup`].
+    #[cfg(any(feature = "test-utils", test))]
```

Comment:
Is this (or any of the functions now hidden behind config flags actually used? I check `with_version` and I couldn't find any place where we actually use it.

Reply:
This is what #645 is meant for.